packages/build/src/plugins/node_version.js

```diff
@@ -17,46 +13,23 @@ const MINIMUM_NODE_VERSION_SUPPORTED = '10.18.0'
 // supported version (Node v10). Else default to the system Node version.
 // Local builds use user's preferred Node.js version.
 // Other plugins use `@netlify/build` Node.js version.
-const addPluginsNodeVersion = function ({ pluginsOptions, mode, nodePath, userNodeVersion, featureFlags }) {
+const addPluginsNodeVersion = function ({ pluginsOptions, mode, nodePath, userNodeVersion }) {
   const currentNodeVersion = cleanVersion(currentVersion)
   return pluginsOptions.map((pluginOptions) =>
-    addPluginNodeVersion({ pluginOptions, currentNodeVersion, userNodeVersion, mode, nodePath, featureFlags }),
+    addPluginNodeVersion({ pluginOptions, currentNodeVersion, userNodeVersion, mode, nodePath }),
   )
 }
 
-// We want to log a warning message for users relying on local or package.json installed plugins whose Node.js
-// versions are lower than the system version we're currently relying (v12). This is part of our effort to decouple
-// the Node.js versions our build system supports and the Node.js versions @netlify/build supports -
-// https://github.com/netlify/pod-workflow/issues/219
-const checkForOldNodeVersions = function ({ pluginsOptions, userNodeVersion, logs, mode }) {
-  if (mode !== 'buildbot' || satisfies(userNodeVersion, `>=${MINIMUM_NODE_VERSION_SUPPORTED}`)) return
-
-  const affectedPlugins = pluginsOptions
-    // `expectedVersion` is only undefined when the plugin is not coming from our plugins directory, those are the cases
-    // where we want to log the warning
-    .filter(
-      ({ loadedFrom, expectedVersion }) =>
-        (loadedFrom === 'local' || loadedFrom === 'package.json') && expectedVersion === undefined,
-    )
-    .map(({ packageName }) => packageName)
-
-  if (affectedPlugins.length === 0) return
-
-  const currentNodeVersion = cleanVersion(currentVersion)
-  logPluginNodeVersionWarning({ logs, pluginNames: affectedPlugins, userNodeVersion, currentNodeVersion })
-}
-
```

Comment:
Log message mentioned in - https://github.com/netlify/build/pull/3311/files#r674171145